docs/DeveloperGuide.md

```diff
@@ -274,6 +278,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
       
+* 1d. User forgets to indicate the priority of the client. 
```

Comment:
Perhaps we can say "does not" instead of forgets

Reply:
oh true my bad